baxter_description/urdf/baxter.urdf

```diff
@@ -1,1558 +0,0 @@
-<?xml version="1.0" ?>
```

Comment:
I would prefer not to delete this file entirely, but rather update it to display the output of the most recent xacro file.